src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Layout, Menu } from 'antd';
+import { Link } from 'react-router-dom';
+
+import {
+  ClockCircleFilled,
+  HomeFilled,
+  SafetyCertificateFilled,
+  DatabaseFilled,
+  BellFilled,
+  GlobalOutlined,
+} from '@ant-design/icons';
+
+const { Sider } = Layout;
+
+const VerticalMenu = () => {
+  return (
+    <Sider width={200}>
+      <Menu
+        mode="inline"
+        defaultSelectedKeys={['1']}
+        defaultOpenKeys={['sub1']}
+        style={{ height: '100%', borderRight: 0 }}
+      >
+        <Menu.Item key="1" icon={<GlobalOutlined />}>
+          GetInspired
+        </Menu.Item>
+        <Menu.Item key="2" icon={<HomeFilled />}>
+          <Link to="/Dashboard">Dashboard</Link>
```

Comment:
[nit] urls should be in lowercase even though it technically doesn't make a difference in the long term